### PR TITLE
editoast: remove fast_rolling_stock fixture (test isolation)

### LINKED ISSUE #5580: editoast: remove fast_rolling_stock fixture (test isolation)
### Description and goal

Part of https://github.com/osrd-project/osrd/issues/5441

### Acceptance criteria

Replace fast_rolling_stock with named variant

### PR DESCRIPTION
**Pull request by shenriotpro**

### CODE REVIEW

**Comment by shenriotpro** [comment]
closes https://github.com/osrd-project/osrd/issues/5580

File: editoast/src/models/pathfinding.rs [hunk 1]
```diff
@@ -37,6 +37,7 @@ use serde::{Deserialize, Serialize};
 #[model(retrieve, delete)]
 #[diesel(table_name = pathfinding)]
 pub struct Pathfinding {
+    // TODO: we probably want the id to be optional and avoid default 0 (test isolation, the DB assigns a value anyway)
```
Comment (Khoyo) [inline editoast/src/models/pathfinding.rs]: No, we ahould upgrade it to the model v2 :) We should avoid false optionals in types retrieved from the db

Comment (shenriotpro) [reply editoast/src/models/pathfinding.rs]: Thanks, how can I get more context on this?

Comment (flomonster) [reply editoast/src/models/pathfinding.rs]: Checkout https://github.com/osrd-project/osrd/pull/5033. I hope it will be merged soon.

**Review by flomonster — APPROVED** [review]
LGMT (reviewed + tested)

**Pull request merged**